deployment/GI01SUMO_Deploy.csv

```diff
@@ -251,3 +251,53 @@ AR30-03,,,,GI01SUMO-SBD12-04-PCO2AA000,5,1,2018-06-08T17:21:00,,CGMGI-01SUMO-000
+AR35-05,,,,GI01SUMO-RII11-02-CTDMOR018,6,1,,,CGMGI-01SUMO-00006,,CGINS-CTDMOR-13375,,,,750,2668,
+AR35-05,,,,GI01SUMO-RII11-02-CTDMOR019,6,1,,,CGMGI-01SUMO-00006,,CGINS-CTDMOR-13376,,,,1000,2668,
+AR35-05,,,,GI01SUMO-RII11-02-CTDMOR020,6,1,,,CGMGI-01SUMO-00006,,CGINS-CTDMOR-13377,,,,1500,2668,
+AR35-05,,,,GI01SUMO-RII11-02-DOSTAD031,6,1,,,CGMGI-01SUMO-00006,,CGINS-DOSTAD-00427,,,,40,2668,
+AR35-05,,,,GI01SUMO-RII11-02-DOSTAD032,6,1,,,CGMGI-01SUMO-00006,,CGINS-DOSTAD-00461,,,,80,2668,
+AR35-05,,,,GI01SUMO-RII11-02-DOSTAD033,6,1,,,CGMGI-01SUMO-00006,,CGINS-DOSTAD-00462,,,,130,2668,
+AR35-05,,,,GI01SUMO-RII11-02-FLORDG031,6,1,,,CGMGI-01SUMO-00006,,CGINS-FLORDG-03418,,,,40,2668,
+AR35-05,,,,GI01SUMO-RII11-02-FLORDG032,6,1,,,CGMGI-01SUMO-00006,,CGINS-FLORDG-03969,,,,80,2668,
+AR35-05,,,,GI01SUMO-RII11-02-FLORDG033,6,1,,,CGMGI-01SUMO-00006,,CGINS-FLORDG-03970,,,,130,2668,
+AR35-05,,,,GI01SUMO-RII11-02-PCO2WC051,6,1,,,CGMGI-01SUMO-00006,,CGINS-PCO2WC-C0105,,,,40,2668,
+AR35-05,,,,GI01SUMO-RII11-02-PCO2WC052,6,1,,,CGMGI-01SUMO-00006,,CGINS-PCO2WC-C0121,,,,80,2668,
+AR35-05,,,,GI01SUMO-RII11-02-PCO2WC053,6,1,,,CGMGI-01SUMO-00006,,CGINS-PCO2WC-C0122,,,,130,2668,
+AR35-05,,,,GI01SUMO-RII11-02-PHSENE041,6,1,,,CGMGI-01SUMO-00006,,CGINS-PHSENE-P0138,,,,20,2668,
+AR35-05,,,,GI01SUMO-RII11-02-PHSENE042,6,1,,,CGMGI-01SUMO-00006,,CGINS-PHSENE-P0139,,,,100,2668,
+AR35-05,,,,GI01SUMO-SBC11-00-CPMENG000,6,1,,,CGMGI-01SUMO-00006,,CGCON-BCPM01-50009,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD11-00-DCLENG000,6,1,,,CGMGI-01SUMO-00006,,CGCON-BDCL11-50009,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD11-01-MOPAK0000,6,1,,,CGMGI-01SUMO-00006,,CGINS-MOPAK0-43264,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD11-03-HYDGN0000,6,1,,,CGMGI-01SUMO-00006,,CGINS-HYDGN0-49891,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD11-04-DOSTAD000,6,1,,,CGMGI-01SUMO-00006,,CGINS-DOSTAD-00127,,,,1,2668,
+AR35-05,,,,GI01SUMO-SBD11-05-SPKIRB000,6,1,,,CGMGI-01SUMO-00006,,CGINS-SPKIRB-00238,,,,-5,2668,
+AR35-05,,,,GI01SUMO-SBD11-06-METBKA000,6,1,,,CGMGI-01SUMO-00006,,CGINS-METLGR-00027,,,,-5,2668,
+AR35-05,,,,GI01SUMO-SBD11-08-NUTNRB000,6,1,,,CGMGI-01SUMO-00006,,CGINS-NUTNRB-01092,,,,1,2668,
+AR35-05,,,,GI01SUMO-SBD12-00-DCLENG000,6,1,,,CGMGI-01SUMO-00006,,CGCON-BDCL12-50009,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD12-01-OPTAAD000,6,1,,,CGMGI-01SUMO-00006,,CGINS-OPTAAD-00187,,,,1,2668,
+AR35-05,,,,GI01SUMO-SBD12-02-FLORTD000,6,1,,,CGMGI-01SUMO-00006,,CGINS-FLORTD-01102,,,,1,2668,
+AR35-05,,,,GI01SUMO-SBD12-03-HYDGN0000,6,1,,,CGMGI-01SUMO-00006,,CGINS-HYDGN0-49876,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,,,CGMGI-01SUMO-00006,,CGINS-PCO2AA-35262,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD12-05-WAVSSA000,6,1,,,CGMGI-01SUMO-00006,,CGINS-WAVSSA-06301,,,,0,2668,
+AR35-05,,,,GI01SUMO-SBD12-06-METBKA000,6,1,,,CGMGI-01SUMO-00006,,CGINS-METLGR-00036,,,,-5,2668,
+AR35-05,,,,GI01SUMO-SBD12-08-FDCHPA000,6,1,,,CGMGI-01SUMO-00006,,CGINS-FDCHPA-64403,,,,-5,2668,

```

Comment:
Looks good.  We probably don't need to put the water depth, since we are not putting lat/lon (for future reference).  We need to be sure we update those with the deployed water depth.